src/Bart/GitHook/CodeFreeze.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Created by IntelliJ IDEA.
+ * User: bvanevery
+ * Date: 2/10/16
+ * Time: 9:02 AM
+ */
+
+namespace Bart\GitHook;
+
+use Bart\Diesel;
+use Bart\Git\Commit;
+
+/**
+ * Prevent changes when code is frozen
+ */
+class CodeFreeze extends GitHookAction
+{
+    /** @var GitHookSystemConfig */
+    private $config;
+
+    public function  __construct()
+    {
+        $this->config = Diesel::create('\Bart\GitHook\GitHookSystemConfig');
```

Comment:
True. I'll wait until that happens before pulling it up. I prefer to not saturate signatures until necessary.